duniverse.opam

```diff
@@ -22,9 +22,6 @@ depends: [
   "opam-core"
   "alcotest" {with-test}
 ]
-pin-depends: [
-  ["dune.dev" "git+https://github.com/NathanReb/dune.git#vendor-stanza"]
```

Comment:
Could you just point this to "git+https://github.com/dune/dune.git for now? That'll keep CI working until the release